scene/3d/audio_stream_player_3d.h

```diff
@@ -68,7 +69,8 @@ class AudioStreamPlayer3D : public Node3D {
 
 	};
 
-	Ref<AudioStreamPlayback> stream_playback;
+	Mutex playback_list_mutex;
```

Comment:
I am not entirely sure why this mutex is needed, as this class is not intended to be used from multiple threads.

Reply:
I'd prefer to keep it to protect against simultaneous access from the physics thread. I don't think it's a huge issue but I do believe it's "technically" necessary. We can remove it and take a wait and see approach for any thread safety issues.

Reply:
While physics uses threads, it does synchronization on the main thread, so it should not be a problem, The idea after Godot 4.0 is to discuss how we can actually make the scene system work better as threaded though, so we will have to see how to better implement these kind of things.

Reply:
Good to know. Removed the mutexes.